test/integration/targets/setup_opennebula/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+opennebula_test:
+  hosts:
+    - hv1
+    - hv2
```

Comment:
Why do you test with two hosts, isn't one enough?

Reply:
yes, one is enough. It works if you define only one too.
You could also test with many if you wanted to do some kind of "stress" test.
the tests will just run on all the hosts that you list there.

Reply:
Thanks for the feedback, but I actually prefer to test with more than 1 host.
I need to illustrate how to organize fixture data when you run test on multiple items: hosts, vms, images, etc. I expect many other tests to be based on this ones.
However, if you want to run your copy against just one host, it works perfectly.

Reply:
I wasn't thinking about my testing, I'm talking about the tests run by Shippable, no need for them to run on two hosts. Let's make them as fast as possible, they are slow anyway.

Reply:
from fixitures is very fast, it is all in memory. have you tried it?

Reply:
I haven't yet, but if it's fast, all is good.